### PR TITLE
updating maven plugin version

### PR DESCRIPTION
**Pull request by Akyiaa**
Signed-off-by: Fiona Ampofo <64271621+Akyiaa@users.noreply.github.com>

### CODE REVIEW

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-integratedtests-pr-192-tttwt for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-integratedtests-pr-192-jgshw for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-integratedtests-pr-192-96ddw for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-integratedtests-pr-192-5vktl for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-integratedtests-pr-192-ndxk5 for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

File: build-locally.sh [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    
+    context=inttests/galasa-inttests-parent
+
+    cmd="gradle --build-cache --parallel \
+    ${CONSOLE_FLAG} \
+    --warning-mode=all \
+    -Dorg.gradle.java.home=${JAVA_HOME} \
+    -PsourceMaven=https://development.galasa.dev/main/maven-repo/managers ${OPTIONAL_DEBUG_FLAG} \
+    -PcentralMaven=https://repo.maven.apache.org/maven2/ \
+    -PtargetMaven=/workspace/git/${context}/inttests/repo \
```
Comment (jadecarino) [inline build-locally.sh:130]: We can remove the targetMaven line here as that's only required for the remote builds :)

**Review by jadecarino — CHANGES_REQUESTED** [review]

**Review by jadecarino — APPROVED** [review]

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**